src/rustc.rs

```diff
@@ -265,7 +265,7 @@ impl QualifiedToolchain {
             Ok(_) | Err(_) if config.custom_toolchain() => {
                 QualifiedToolchain::custom(toolchain, &sysroot, config, msg_info)
             }
-            Ok(_) => return Err(eyre::eyre!("toolchain is not fully qualified")
```

Comment:
Just a clippy lint that was detected with the latest nighties, and completely unrelated to the PR.

Reply:
If we move this to cross-toolchains, then this needs to be included in a separate PR.